Apps/Admin/Client/Store/Analytics/AnalyticsActions.cs

```diff
@@ -0,0 +1,155 @@
+    /// The action representing the initiation of comments action load.
+    /// </summary>
+    public class LoadCommentsAction : AnalyticsBaseAction
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LoadCommentsAction"/> class.
+        /// </summary>
+        /// <param name="startDate">Optional start date to include in the query.</param>
+        /// <param name="endDate">Optional end date to include in the query.</param>
+        public LoadCommentsAction(DateTime? startDate, DateTime? endDate)
+            : base(startDate, endDate)
+        {
+        }
+    }
+
+    /// <summary>
+    /// The action representing the initiation of notes action load.
+    /// </summary>
+    public class LoadNotesAction : AnalyticsBaseAction
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LoadNotesAction"/> class.
+        /// </summary>
+        /// <param name="startDate">Optional start date to include in the query.</param>
+        /// <param name="endDate">Optional end date to include in the query.</param>
+        public LoadNotesAction(DateTime? startDate, DateTime? endDate)
+            : base(startDate, endDate)
+        {
+        }
+    }
+
+    /// <summary>
+    /// The action representing the initiation of ratings load.
+    /// </summary>
+    public class LoadRatingsAction : AnalyticsBaseAction
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LoadRatingsAction"/> class.
+        /// </summary>
+        /// <param name="startDate">Optional start date to include in the query.</param>
+        /// <param name="endDate">Optional end date to include in the query.</param>
+        public LoadRatingsAction(DateTime? startDate, DateTime? endDate)
+            : base(startDate, endDate)
+        {
+        }
+    }
+
+    /// <summary>
+    /// The action representing the initiation of inactive users load.
+    /// </summary>
+    public class LoadInactiveUsersAction
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LoadInactiveUsersAction"/> class.
+        /// </summary>
+        /// <param name="inactiveDays">The days inactive to filter the users last login.</param>
+        /// <param name="timeOffset">The offset from the client browser to UTC.</param>
+        public LoadInactiveUsersAction(int inactiveDays, int timeOffset)
+        {
+            this.InactiveDays = inactiveDays;
+            this.TimeOffset = timeOffset;
+        }
+
+        /// <summary>
+        /// Gets or sets inactive days.
+        /// </summary>
+        public int InactiveDays { get; set; }
+
+        /// <summary>
+        /// Gets or sets time offset.
+        /// </summary>
+        public int TimeOffset { get; set; }
+    }
+
+    /// <summary>
+    /// The action representing a successfull action load.
+    /// </summary>
+    public class LoadSuccessAnalyticsAction : BaseLoadSuccessAction<HttpContent>
```

Comment:
Nit: "LoadSuccessAnalyticsAction" should be "LoadSuccessAction", since it is contained within "AnalyticsActions".